src/shell/components/FontAwesomeIcon.tsx

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+import React = require('react');
+
+export class FontAwesomeIcon extends React.Component<{ path?: string }, {}> {
```

Comment:
Instead of calling this FontAwesomeIcon, can we call this something like `ShellIcon`? My thinking is that we will be getting out own icons in time and instead of needing to rename post that we can just update the `paths`.